CreatePerson/function.json

```
@@ -0,0 +1,25 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "CreatePerson",
+      "methods": ["post"],
+      "route": "people"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "queue",
+      "direction": "out",
+      "name": "bothIds",
```

How about giving this a name that represents not what it receives, but what it symbolises.

```suggestion
      "name": "addPersonToGroupQueue",
```